tc-package.sh

```diff
@@ -9,6 +9,15 @@ mkdir -p ${TASKCLUSTER_ARTIFACTS} || true
 cp ${DS_ROOT_TASK}/DeepSpeech/tf/bazel-bin/tensorflow/libtensorflow_cc.so ${TASKCLUSTER_ARTIFACTS}
 cp ${DS_ROOT_TASK}/DeepSpeech/tf/bazel-bin/tensorflow/tools/graph_transforms/transform_graph ${TASKCLUSTER_ARTIFACTS}
 
-artifact_root_dir=$(dirname "${DS_ROOT_TASK}")
-artifact_tar_dir=$(basename "${DS_ROOT_TASK}")
-tar -C ${artifact_root_dir} -cf - ${artifact_tar_dir} | pixz -9 > ${TASKCLUSTER_ARTIFACTS}/home.tar.xz
+# Let's perform some cleanup
+rm -fr ${DS_ROOT_TASK}/dls/
+
+# Keeping homebrew seems to screw later, so let's just remove it and save some space
+if [ "${OS}" = "Darwin" ]; then
+    rm -fr ${TASKCLUSTER_TASK_DIR}/homebrew/
+fi;
+
+# Make a tar of
+#  - /home/build-user/ (linux
+#  - /Users/build-user/TaskCluster/HeavyTasks/X/ (OSX)
+tar -C ${DS_ROOT_TASK} -cf - . | pixz -9 > ${TASKCLUSTER_ARTIFACTS}/home.tar.xz
```

Comment:
We switch back to what was done at first, since we do not need to have the "task-SHA1" stuff. It will need to be handled in parity on the DeepSpeech side (WIP in https://github.com/mozilla/DeepSpeech/pull/842)